### ISSUE TITLE
scale-free arrow

### ISSUE THREAD

**Issue by Datseris** [JuliaPlots/Plots.jl#778]
The possibility of creating an arrow that does not change scale when zooming into the plot is missing here.

In PyPlot this is achieved with the `quiver` plot, using the settings:
```
q1 = quiver(location, velocity, angles = "xy", scale = 1)
```
Can a similar approach become available for `Plots`?

**Comment by mkborregaard** [comment]
Have you tried something like `quiver(x, y, velocity = v)`

**Comment by Datseris** [comment]
Wait, why isn't my comment here? I have commented 5 days ago!!! 

Anyway, my comment was: with your suggestion, when you zoom into a figure, the arrow becomes bigger, i.e. it is not *scale free*. I want it so that when I zoom into the figure, the arrow **does not become bigger**.

Probably, there has to be a way to make the arrow scale depend on the axis and not the data units...

**Comment by mkborregaard** [comment]
Do you mean you don't want the arrow to become longer? I thought you meant the arrowhead.

**Comment by Datseris** [comment]
Yes, I don't want the arrow to become longer (or thicker or bigger). 
Of course the same goes for the arrowhead as well, it also shouldn't change size!

And very important: The *orientation* of the arrow has to be correct. That is why on the `PyPlot` command, one gives `angles = "xy", scale = 1)` . This means that the arrow orientation comes from the data, but the scale does not

**Comment by mkborregaard** [comment]
OK, right.
If someone would be interested in implementing this it would require a little thinking on how to define the angle, length etc of the vectors so that it is consistent with Plots syntax (i.e. it shouldn't be necessary to add any extra keywords to get this functionality).

**Comment by ChrisRackauckas** [comment]
I think this issue really just boils down to the whole "scale-free" DPI / relative scaling issue, right?
